pytket/CMakeLists.txt

```diff
@@ -38,31 +38,82 @@ else()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Werror")
 endif()
 
-function(build_module module)
-    pybind11_add_module(${module} ${ARGN})
-    target_include_directories(${module} PRIVATE binders/include)
-    target_link_libraries(${module} PRIVATE ${CONAN_LIBS})
-endfunction(build_module)
-
-build_module(circuit
+pybind11_add_module(circuit
     binders/circuit/main.cpp
     binders/circuit/unitid.cpp
     binders/circuit/boxes.cpp
     binders/circuit/classical.cpp
     binders/circuit/Circuit/main.cpp
     binders/circuit/Circuit/add_op.cpp
     binders/circuit/Circuit/add_classical_op.cpp)
-build_module(routing binders/routing.cpp)
-build_module(transform binders/transform.cpp)
-build_module(predicates binders/predicates.cpp)
-build_module(passes binders/passes.cpp)
-build_module(program binders/program.cpp)
-build_module(partition binders/partition.cpp)
-build_module(pauli binders/pauli.cpp)
-build_module(logging binders/logging.cpp)
-build_module(utils_serialization binders/utils_serialization.cpp)
-build_module(tailoring binders/tailoring.cpp)
-build_module(tableau binders/tableau.cpp)
-build_module(zx
+target_include_directories(circuit PRIVATE binders/include)
```

Comment:
Have you thought about naming all the python packages something like `pytket-circuit` to make clear that they are self defined?

Reply:
This would mean changing the names of the Python modules we expose, e.g. `pytket.circuit` would become `pytket.pytket-circuit`.

At least the names are lower-case so not identical to the tket component names.